### ISSUE TITLE
100 companies milestone!

### ISSUE THREAD

**Issue by alexpdp7** [remote-es/remotes#54]
Someone noted we've reached 100 companies listed ( https://github.com/remote-es/remotes/issues/11#issuecomment-821976942 ), including Manfred.

I've done a quick review of the listing and I haven't found many broken links (some were outdated, but I managed to find updated URLs easily: https://github.com/remote-es/remotes/commit/269d0d33bafe66b69ca606cd78ffcef151af7f13 )- I need to check two companies that have been acquired to get some clarification on them, see #53 .

However, at some point maybe we should re-consider our current format. It might be a bit uncomfortable to open 100 links (although I just did that for the review). From the last time we discussed that, more than a year ago, GitHub has added GitHub Discussions, which might be a way to handle job postings.

Thoughts @acroca @uesteibar ? I think this might be worth thinking about- I don't think any next step would be easy, so maybe we want to stay as-is for a while more? Perhaps it's worth reaching out to some communities for discussion (BcnEng?)...

I'm taking this easy for now, so I'll just ping you and see if you guys have any input.

**Comment by eduherminio** [comment]
I would find a 'technologies' column particularly useful 😃

**Comment by uesteibar** [comment]
unpopular probably but: CSV files are sortable and filterable on github right? see [this docs](https://docs.github.com/en/github/managing-files-in-a-repository/rendering-csv-and-tsv-data).

**Comment by alexpdp7** [comment]
Given that we are having discussions about requirements for listings, etc. I started to do some data collection to see our current status. See:

https://docs.google.com/spreadsheets/d/116xrZK2te2TSGfxbDaAKAsgdu8AS0IifnwSj9ZlM3Ag/edit?usp=sharing

I will try to get this finished so we can make decisions based on what we have there.

**Comment by alexpdp7** [comment]
Listing is now complete (and I've cleaned a few URLs and removed listings), so we can decide whether to be more "strict" in our rules, while knowing how many companies we would be removing, etc.

@uesteibar @acroca I haven't looked at the numbers yet. We are on the edge- with 108 companies in the listing I think we can afford being a little bit more strict, I think. I'm *slightly* partial to not being strict, but I welcome your opinions on which kinds of restrictions could we add to make the listing more useful.

**Comment by uesteibar** [comment]
I'd say the following:
- if there's no clear indication in the list _but_ a clarifying note can be provided, I'd add that next to the company (e.g. _all IT positions are remote)
- If only some positions are remote and there's no way to differentiate them (aka it's arbitrary, e.g _not all IT positions are remote), then I'd suggest we require clarification in the job list

**Comment by alexpdp7** [comment]
Yeah, I'm dumb- I hadn't thought of this. Using the spreadsheet we can add some clarifying "tags" in bulk and request new entries to have them if required.

I can give it a shot if I have some spare time.

**Comment by alexpdp7** [comment]
I tried https://github.com/remote-es/remotes/pull/65 , @acroca @uesteibar , what do you think? (We can discuss in the PR itself).

This would add some "required tagging policy", which may help?

**Comment by alexpdp7** [comment]
Heh, just realized we are up to 204 companies, nearly 1k stars. Probably the listing is becoming big enough to be "less reasonable" to "open all in tabs". Some of these companies look possibly "gone" (see #116 , I kept some companies for which I had no evidence they were gone- they only had 0 offers on their listings). So perhaps it's time to rethink this once again.
